SimPEG/electromagnetics/natural_source/receivers.py

```diff
@@ -29,6 +31,8 @@ class BaseRxNSEM_Point(BaseRx):
         {
             "real": ["re", "in-phase", "in phase"],
             "imag": ["imaginary", "im", "out-of-phase", "out of phase"],
+            "apparent_resistivity": ["app_rho"],
```

Comment:
here, for consistency, I would suggest we consider 
`"apparent resistivity" : ["apparent_resistivity", "app_rho"]`

(e.g. have the default be with the space). What do you think of that?